media-libs/libinstpatch/libinstpatch-1.1.3.recipe

```diff
@@ -71,6 +71,15 @@ INSTALL()
 	prepareInstalledDevelLib libinstpatch-1.0
 	fixPkgconfig
 
+# fix fixPkgconfig
+	if [ -n "$secondaryArchSuffix" ]; then
+		sed -i 's,\/headers/x86,\/headers/x86/libinstpatch-2,g' \
+			$prefix/$relativeDevelopLibDir/pkgconfig/libinstpatch-1.0.pc
+		else
+		sed -i 's,\/headers,\/headers/libinstpatch-2,g' \
+			$prefix/$relativeDevelopLibDir/pkgconfig/libinstpatch-1.0.pc
+	fi
```

Comment:
This can be simplified using `$secondaryArchSubDir`:

```suggestion
	sed -i "s,\/headers$secondaryArchSubDir,\/headers$secondaryArchSubDir/libinstpatch-2,g" \
		$developLibDir/pkgconfig/libinstpatch-1.0.pc
```

Reply:
Thanks! Should update that one to my wiki too :D